src/openrct2/entity/Peep.h

```diff
@@ -352,13 +352,13 @@ struct Peep : EntityBase
     };
     // Normally 0, 1 for carrying sliding board on spiral slide ride, 2 for carrying lawn mower
     uint8_t SpecialSprite;
-    PeepActionSpriteType ActionSpriteType;
-    // Seems to be used like a local variable, as it's always set before calling SwitchNextActionSpriteType, which
+    PeepAnimationType AnimationType;
```

Comment:
If we’re renaming fields, we may as well change these to be in `lowerCamelCase,` as we adopted earlier this year.

Reply:
It's proving to be a little painful, so I suggest we tackle the style change one entire struct at a time, indeed.